src/ripple/app/misc/Validations.cpp

```diff
@@ -387,6 +387,31 @@ class ValidationsImp : public Validations
         return ret;
     }
 
+    std::uint32_t getValidationTime (uint256 const& hash, int minValidations)
+    {
+        if (minValidations < 1)
+            minValidations = 1;
+
+        std::vector <std::uint32_t> times;
+
+        {
+            ScopedLockType sl (mLock);
+
+            auto j = findSet (hash);
+            for (auto& it : *j)
```

Comment:
@scottschurr I don't see how `times` can possibly be sorted because `findSet` returns an `unordered_map`. Plus what you said.